client/src/pages/Search.jsx

```diff
@@ -14,7 +14,7 @@ export default function Search() {
     const options = {
       method: "GET",
       headers: {
-        "X-RapidAPI-Key": process.env.REACT_APP_APIKEY,
```

Comment:
Please put this back how it was- its part of the deployed app.

Reply:
looks like it was refactored to be handled by the backend 
nice play